### PR TITLE
Updating UI for Profile, Login and Register components

### PR DESCRIPTION
**Pull request by thvphuc246**
This request replaces PR #3 with further updates in UI components (Profile, Login, Register).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "user-management",
-  "version": "0.0.0",
+  "version": "1.0.0",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build",
+    "build": "ng build --prod",
```
Comment (beannguyen) [inline package.json]: I prefer to use:

`ng build --prod --aot --buildOptimizer --commonChunk --vendorChunk --optimization --progress`

To undestand AOT: https://angular.io/guide/aot-compiler

File: src/app/profile/profile.component.ts [hunk 2]
```diff
@@ -6,14 +6,37 @@ import { TokenStorageService } from '../_services/token-storage.service';
...
 
   ngOnInit(): void {
-    this.currentUser = this.token.getUser();
+    // this.currentUser = this.token.getUser();
+    this.currentUser = this.demoUser;
   }
 
 }
+
+export class User {
```
Comment (beannguyen) [inline src/app/profile/profile.component.ts]: Move view model classes to a folder: "models" for ex.
I do not prefer we declare the class inside the components.

Comment (thvphuc246) [reply src/app/profile/profile.component.ts]: Yes this is just for the first time of UI testing. There will be more model classes to be implemented, and I will put them in a separate directory later.

**Review by voquanghien — APPROVED** [review]
Please make sure you will always share your problems with us if you cannot decide or execute because our business is continuously running with time. @thvphuc246

File: src/app/profile/profile.component.ts (hunk 2 above)
Comment (beannguyen) [reply src/app/profile/profile.component.ts]: Great

**Review by beannguyen — APPROVED** [review]

**Pull request merged**